src/index.js

```diff
@@ -1,2 +1,57 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render(num) {
+  const element = (
+    <div id="hello" className="greeting">
+      <p>Hello world!</p>
+      <p>
+        <button
+          type="button"
+          onClick={() => {
+            render(num + 1);
+          }}
```

Comment:
```suggestion
          onClick={() => render(num + 1)}
```
거의 코드의 차이는 없는데 어떤 코드가 더 읽기 쉬운 것 같나요?

불필요한 줄 바꿈을 지양하는게 좋아요.

코드를 작성할 때 줄바꿈이나 공백 하나에도 다 의미단위로 의도를 생각하시면서 코드를 작성하시면 훨씬 가독성 좋은 코드를 작성하실 수 있어요!